data/os/Suse.yaml

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
I am just trusting that this works, rather than deeply reviewing that these changes are correct.

I'm deferring to security vetting to see if they see issues.